pkg/sdk/metrics.go

```diff
@@ -0,0 +1,28 @@
+package sdk
+
+import (
+	"net/http"
+	"strconv"
+
+	k8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
As a suggestion, the [prometheus OpenCensus exorter library](https://godoc.org/go.opencensus.io/exporter/prometheus) could be used.

Here is a code snippet to illustrate the usage:
```golang
import (
        "github.com/sirupsen/logrus"
        "github.com/prometheus/client_golang/prometheus"
        ocprometheus "go.opencensus.io/exporter/prometheus"
	ocview "go.opencensus.io/stats/view"
)

metricsRegistry := prometheus.NewRegistry()
metricsRegistry.MustRegister(prometheus.NewGoCollector())

prometheusExporter, err := ocprometheus.NewExporter(ocprometheus.Options{
	Namespace: "operator_sdk",
	Registry:  metricsRegistry,
})
if err != nil {
        logrus.Fatalf("Failed to create prometheus exporter: %v", err)
	return
}

ocview.RegisterExporter(prometheusExporter)

http.Handle("/"+k8sutil.PrometheusMetricsPortName, prometheusExporter)
```

IMHO, this project should integrate and use the [OpenCensus](https://opencensus.io/) library.

Reply:
@secat I'm not too familiar with the OpenCensus library so I can't comment on the need for it yet. However this PR is only meant to be the starting point for exposing the Prometheus metrics so I would prefer to keep it simple right now and iterate on more changes after this.
It'll be helpful if you can create an issue so we can discuss the use cases for the OpenCensus library in the SDK.

Reply:
@hasbro17 I have created the issue #343